REDIRECT.en.md

```diff
@@ -0,0 +1,211 @@
+
+In the case of the Catalogue, it is necessary to refer to the [Italy folder of the w3id GIT](https://github.com/perma-id/w3id.org/tree/master/italia), in which the subfolders MUST be added, each for a particular thematic area, which will contain the redirection files, possibly organized in further subfolders, and the `README.md` file. The addition of the `'/Italy'` subfolders will be subject to approval by the Italy Committee; subsequently, they will be managed independently and with direct interfacing between w3id.org and the contacts indicated in the `README.md`. The folder and its subfolders, `.htaccess` files, and `README.md` files MUST be created on the [w3id git repository](https://github.com/perma-id/w3id.org) by the Contributor.
+
+## Publishing htaccess on w3id
+
+### 1. fork of the w3id.org GIT repository
+
+The first step for registering the redirects is the local fork of the
+[w3id GIT Italy folder](https://github.com/perma-id/w3id.org/tree/master/italia). The
+permanent identifiers (URI) will be defined based on the path in which the various `htaccess` files will be inserted.
+In this case, the path to the "root" folder for folder-name must be `"italia/<folder-name>/"`, therefore the namespace of the URIs defined in it will be `w3id.org/italia/<folder-name>/`.
+
+### 2. adding the folder
+
+In the local repository created starting from the fork, it will be necessary to create the `folder-name` folder which will contain the tree of subfolders and the related `.htaccess` files, in addition to the `README.md`.
+
+Below is an example of a folder tree under `"/italia"`:
+
+<pre>
+italia
+|--folder-name
+|   |--controlled-vocabulary
+|   |   |-.htaccess
+|   |--data
+|   |   |-.htaccess
+|   |--onto
+|   |   |-.htaccess
+|   |README.md
+</pre>
+
+This will define the following URIs:
+-	`w3id.org/italia/<folder-name>/controlled-vocabulary`
+-	`w3id.org/italia/<folder-name>/data`
+-	`w3id.org/italia/<folder-name>/onto`
+
+The `<folder-name>` is very important as it MUST be placed in specific parameters described later in this document, and will always be used to refer to the Contributor's set of semantic resources as part of the redirect file configuration.
+
+The redirect rules associated with each uri MUST be defined in the relevant `.htaccess` files described below. The `README.md` file MUST contain the names of the contacts along with their email and github references. These references MUST take care of the management of the folder and related files following the approval of "Italia". You SHOULD take the `README.md` file under the `"/italia"` folder as an example.
+
+### 3. creating the pull request
+
+Once the GIT repository has been modified locally, a pull request MUST be created, which will be analyzed and possibly validated by "Italia"; in the pull request the contacts present in the [README.md file under “/italia”](https://github.com/perma-id/w3id.org/blob/master/italia/readme.md) MUST be indicated as reviewers.
```

Comment:
```suggestion
Once the GIT repository has been modified locally, a pull request MUST be created, which will be analyzed and possibly validated by "Italia"; in the pull request the contacts present in the [README.md file under `/italia`](https://github.com/perma-id/w3id.org/blob/master/italia/readme.md) MUST be indicated as reviewers.
```

Here and elsewhere, don't quote `preformatted` strings unless you want the `"` to be in the folder name ;)

Reply:
The last commit fixes this.